viziphant/array_data.py

```diff
@@ -0,0 +1,163 @@
+"""
+This module contains functions to plot activity data from two-dimensional
+recording arrays.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import neo
+
```

Comment:
Quantities package is used in doctest
```suggestion
import quantities as pq
```

Reply:
So for now, I put the quantities library in the doc test itself, but this should be discussed again.

Reply:
Answering your first question,
we can do `pip install viziphant[test]` or `[all]` if we define the appropriate flag in `setup.py`.
but now I noticed a larger problem - we don't have requirements at all - neither `requirements.txt` nor in `setup.py`.
-_-